.github/workflows/release.yml

```diff
@@ -97,6 +97,27 @@ jobs:
           tar czf "$staging.tar.gz" "$staging"
           echo "ASSET=$staging.tar.gz" >> $GITHUB_ENV
 
+      - name: Create Debian package
+        shell: bash
+        if: startsWith(matrix.job.os, 'ubuntu')
```

Comment:
It looks like cargo-deb supports passing a —target flag to create deb packages for each platform, would it be possible to do that?  Not critical for the first version I’d say, nice to have, we can always extend it later.